config/routes.rb

```diff
@@ -9,23 +9,20 @@
 
   resources :settings, only: %i[show create]
 
-  namespace :admin do
-    root to: "users#index"
-    resources :users
-    resources :articles do
-      post "publish", on: :member
-      post "unpublish", on: :member
-    end
-  end
-
   constraints CmsRouteConstraint.new do
     devise_for :users
     devise_scope :user do
       get "sign_in", to: "devise/sessions#new"
       delete "sign_out", to: "devise/sessions#destroy"
     end
 
-    scope :cms do
+    namespace :admin do
+      root to: "users#index"
+      resources :users
+      resources :articles do
+        post "publish", on: :member
+        post "unpublish", on: :member
+      end
       resources :content_pages, path: "pages" do
```

Comment:
Sorry, just thinking of this now, but can we also add a redirect for each of the endpoints so that :cms goes to :admin ?